src/main/features/claimant-response/views/macro/timelineTable.njk

```diff
@@ -0,0 +1,22 @@
+{% macro timelineFragment(timeline) %}
+  <table>
+    <thead>
+    <tr>
+      <th scope="col">{{ t('Date') }}</th>
+      <th scope="col">{{ t('What happened') }}</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for event in timeline.rows %}
+      <tr>
+        <td>{{ event.date }}</td>
+        <td>{{ event.description }}</td>
+      </tr>
+    {% endfor %}
+    </tbody>
+  </table>
+  {% if timeline.comment %}
+    <h3 class="heading-small form-group-related">{{ t('Why they disagree with your timeline') }}</h3>
```

Comment:
The defendant's input prompt for this data is `Add any comments about their timeline (optional)` - they haven't been instructed specifically to describe why they disagree with the timeline. Can you verify this with POs please?